src/rate_limiter/src/lib.rs

```diff
@@ -495,6 +589,15 @@ mod tests {
         fn get_processed_refill_time(&self) -> u64 {
             self.processed_refill_time
         }
+
+        // After a restore, we cannot be certain that the last_update field has the same value.
+        fn equal_to(&self, other: &TokenBucket) -> bool {
+            (other.capacity() == self.capacity())
+                && (other.one_time_burst() == self.one_time_burst())
+                && (other.refill_time_ms() == self.refill_time_ms())
+                && (other.budget() == self.budget())
+                && (other.last_update >= self.last_update)
```

Comment:
This is a bit weird to be present in an `equal_to` function. How about we implement `PartialEq` instead and ignore that field?

Reply:
`PartialEq` is already derived for `TokenBucket`.  See [comment](https://github.com/firecracker-microvm/firecracker/pull/1749#discussion_r406274757)

Reply:
`equal_to` is misleading though because you are not checking equality. You could still ignore the `last_update` field and call it `partial_eq` or something.

Reply:
Done.